test/uninsured-vesting-v1/uninsured-vesting-v1.test.ts

```diff
@@ -220,89 +221,172 @@ describe("VestingV1", () => {
           await projectToken.amount(0.1)
         );
 
-        await vesting.methods.recoverToken(projectToken.options.address).send({ from: deployer });
+        await vesting.methods.recoverToken(projectToken.options.address).send({ from: daoWallet });
         expect(await projectToken.methods.balanceOf(vesting.options.address).call()).to.be.bignumber.eq(await projectToken.amount(TOKENS_PER_USER));
       });
 
       it("handles zero token balance gracefully", async () => {
         const startingBalance = await someOtherToken.methods.balanceOf(vesting.options.address).call();
         expect(startingBalance).to.be.bignumber.zero;
-        await vesting.methods.recoverToken(someOtherToken.options.address).send({ from: deployer });
+        await vesting.methods.recoverToken(someOtherToken.options.address).send({ from: daoWallet });
         expect(await someOtherToken.methods.balanceOf(vesting.options.address).call()).to.be.bignumber.zero;
       });
     });
 
     describe("access control", () => {
-      it("cannot call activate if not owner", async () => {
-        await expectRevert(async () => vesting.methods.activate(await getDefaultStartTime()).send({ from: anyUser }), "Ownable: caller is not the owner");
+      const generateAccessControlErrorMsg = (account: string, role: string) =>
+        `VM Exception while processing transaction: reverted with reason string 'AccessControl: account ${account.toLowerCase()} is missing role ${role}'`;
+
+      describe("only project", () => {
+        it("can activate", async () => {
+          const projectRole = await vesting.methods.PROJECT_ROLE().call();
+
+          const expectedInvalidUsers = [daoWallet, deployer];
+
+          for (const invalidUser of expectedInvalidUsers) {
+            await expectRevert(
+              async () => vesting.methods.activate(await getDefaultStartTime()).send({ from: invalidUser }),
+              generateAccessControlErrorMsg(invalidUser, projectRole)
+            );
+          }
+
+          await vesting.methods.setAmount(user1, await projectToken.amount(TOKENS_PER_USER)).send({ from: projectWallet });
+          await vesting.methods.activate(await getDefaultStartTime()).send({ from: projectWallet });
+          // should not revert
```

Comment:
nit: to be consistent with our style, would consider separating the failure from success cases to different tests